casa/IO/MultiFile.h

```diff
@@ -113,14 +156,35 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     // Open or create a MultiFile with the given name.
     // Upon creation the block size can be given. If 0, it uses the block size
     // of the file system the file is on.
-    // If useODirect=True, the O_DIRECT flag in used (if supported). It tells the
-    // kernel to bypass its file cache to have more predictable I/O behaviour.
+    // <br>If useODirect=True, the O_DIRECT flag is used (if supported).
+    // It tells the kernel to bypass its file cache to have more predictable
+    // I/O behaviour.
+    // <br>If useCRC=True, 32-bit CRC values are calculated and stored for
```

Comment:
I fully agree, but I could not think of another way.
Do you have a better idea?

Reply:
Maybe have a derived class in the test. I'll look into it.

Reply:
I would say writing to a large file, maybe a dummy file using the memory as output buffer.

Reply:
Removed test code from MultiFile. Instead created a test program containing a class MultiFileLarge derived from MultiFile. Two hooks have been added to MultiFile to make it all possible.
Using memory as a buffer is not possible because it needs to be tested if the file is ok when ending prematurely.